uint16 elbowPos
uint16 shoulderPos
uint16 basePos
